### PR TITLE
add a stream-bytes/:n endpoint & tests

### PR DESCRIPTION
**Pull request by jordan0day**
tried to match the semantics of httpbin's version -- includes support for "seed" and "chunk_size" parameters

### CODE REVIEW

**Comment by edgurgel** [comment]
Can you also add the link to https://github.com/edgurgel/httparrot/blob/master/priv/index.html ?

**Comment by edgurgel** [comment]
Thank you very much!

File: lib/httparrot/stream_bytes_handler.ex [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+  def get_bytes(req, state) do
+    {n, seed, chunk_size} = state
+    :random.seed(seed, seed, seed)
+    {{:chunked, stream_response(n, chunk_size)}, req, nil}
+  end
+
+  defp stream_response(n, chunk_size) do
+    fn(send_func) ->
+      Stream.repeatedly(fn -> :random.uniform(255) end)
```
Comment (edgurgel) [inline lib/httparrot/stream_bytes_handler.ex]: Pretty cool :+1:

**Comment by jordan0day** [comment]
D'oh! Totally forgot to add a link on the index.html page, nice catch. :tada:

**Comment by edgurgel** [comment]
Thanks again! :fireworks:

**Pull request merged**